pkg/grapicmd/di/providers.go

```diff
@@ -90,30 +78,30 @@ func ProvideToolRepository(ctx *grapicmd.Ctx, gexCfg *gex.Config) (tool.Reposito
 	return toolRepo, nil
 }
 
-func ProvideProtocWrapper(ctx *grapicmd.Ctx, ui clui.UI, toolRepo tool.Repository) protoc.Wrapper {
+func ProvideProtocWrapper(ctx *grapicmd.Ctx, ui cli.UI, toolRepo tool.Repository) protoc.Wrapper {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported function ProvideProtocWrapper should have comment or be unexported